platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/LdapClaimsHandler.java

```diff
@@ -112,8 +116,34 @@ public ProcessedClaimCollection retrieveClaimValues(ClaimCollection claims,
 
             connection = connectionFactory.getConnection();
             if (connection != null) {
-                BindResult bindResult = connection.bind(bindUserDN,
-                        bindUserCredentials.toCharArray());
+                BindRequest request;
```

Comment:
An optional way to shorten this a little would be to create a static map in class initialization like this...
```java
            Map<String, Optional<BiFunction<String, char[], BindRequest>>> lookup = ImmutableMap.of(
                    "Simple",
                    Optional.of(Requests::newSimpleBindRequest),
                    "SASL",
                    Optional.of(Requests::newPlainSASLBindRequest),
//...
            );
```
Then replace the whole switch statement with 
```java
                BindRequest request = lookup.get(bindMethod)
                        .orElse(Requests::newSimpleBindRequest)
                        .apply(bindUserDN, bindUserCredentials.toCharArray());
```